src/hooks/useSignMessage/useSignMessage.ts

```diff
@@ -271,11 +279,24 @@ const useSignMessage = ({
 
         await verifySignature(toSign, sig, requestedNetwork?.id)
 
-        resolve({ success: true, result: sig })
         addToast('Successfully signed!')
+
+        setSignedMessages( [...signedMessages, {
+            account: account.id,
```

Comment:
Maybe name this `accountId`, instead of `account`?

It drives me crazy in the web app, when `account` is used sometimes for id (string), sometimes for object (full account).

Reply:
totally Agree!